.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,27 @@
+name: "Linters"
+on:
+  workflow_dispatch:
+  push:
+    branches:
+       - main
+  pull_request:
+
+env:
+  GO_VERSION: "1.17.x"
+  GO_STABLE_VERSION: true
+
+jobs:
+  golangci:
+    name: "GolangCI parkour around go 1.18 🤦"
```

Comment:
we shouldn't need this new workflow, but if we did we would need this check to be aligned with the release quality gate

Reply:
we do because `bouncer` fails on on go1.17 trying to read `runtime/debug`, and it doesn't support exclusion of internal packages. The simplest is to isolate golangci on go1.17